### PR TITLE
add 'is_hr_rule' filtering based on enabled addons

### LINKED ISSUE #777: Enable HR-addon
As a User I want to be able to specify if I am in the context of an "HR power dynamic"\.

<p></p>

Acceptance criteria:

- Enable HR related rules via a toggle in the quick menu

### PR DESCRIPTION
**Pull request by lsmith77**
fixes https://github.com/witty-works/nlp_api/issues/777

### CODE REVIEW

**Review by hellasol — APPROVED** [review]
needs a rebase

File: tests/test_witty_addons/test_api_hr_addon_on/input.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "text": "Wir suchen eine fachlich analytische Arbeitskraft für unser Team.",
+    "config": {
+        "addons": [
+            "hr"
```
Comment (lsmith77) [inline tests/test_witty_addons/test_api_hr_addon_on/input.json:5]: @hellasol HR add-on is enabled

File: tests/test_witty_addons/test_api_hr_addon_off/input.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "text": "Wir suchen eine fachlich analytische Arbeitskraft für unser Team.",
+    "config": {
+        "addons": []
```
Comment (lsmith77) [inline tests/test_witty_addons/test_api_hr_addon_off/input.json:4]: HR add-on is disabled

File: tests/test_witty_addons/test_api_hr_addon_unset/input.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "text": "Wir suchen eine fachlich analytische Arbeitskraft für unser Team."
```
Comment (lsmith77) [inline tests/test_witty_addons/test_api_hr_addon_unset/input.json:2]: no add-ons are set => HR-add-on is enabled

**Pull request merged**